combinations/hash.tsv

```diff
@@ -546,3 +546,4 @@ macs2=2.2.9.1,mawk=1.3.4
 kraken2=2.1.3,coreutils=9.4
 bwa=0.7.17,samtools=1.19.2,sambamba=1.0
 optitype=1.3.5,coincbc=2.10.0
+r-base=4.3.2,r-hwriter=1.3.2,r-ggplot2=3.4.4,r-ini=0.3.1,r-ashr=2.2_63,r-dplyr=1.1.4,r-ggrepel=0.9.5,r-png=0.1_8,bioconductor-reportingtools=2.42.2,bioconductor-deseq2=1.42.0,bioconductor-complexheatmap=2.18.0,bioconductor-tximport=1.30.0
```

Comment:
r-base does not need to be specified.

Reply:
Hi @bgruening, thanks. First time doing this so apologies if my questions are silly, but do I need to remove r-base to proceed, or it was just a comment? Also, when will I know when the mulled container is available to use? Thanks